src/Typography/Typography.types.ts

```diff
@@ -0,0 +1,25 @@
+import { ElementType, HTMLAttributes, ReactNode } from 'react';
+export interface TypographyProps
+  extends HTMLAttributes<HTMLHeadingElement>,
+    HTMLAttributes<HTMLParagraphElement> {
+  /**Applies the theme typography styles.*/
+  variant?:
+    | 'hero'
+    | 'h1'
+    | 'h2'
+    | 'h3'
+    | 'h4'
+    | 'h5'
+    | 'h6'
+    | 'subtitle'
+    | 'body1'
+    | 'body2'
+    | 'caption'
+    | 'disclaimer';
+  /**The content of the component*/
+  children?: ReactNode;
+  /**Set the text-align on the component.*/
+  align?: 'center' | 'inherit' | 'justify' | 'left' | 'right' | 'start' | 'end';
+  /**The component used as the root node of the content. For example `div`, `span`, ...*/
+  component?: ElementType;
```

Comment:
I think component is required. Do we have a reason to make it optional  ?

Reply:
We made an assumption that there is a relation between `component` and `varient`, but I think there is no relation between them.

Reply:
Okay, I changed that as we agreed in our meeting. the variants represents the style only :+1: